core/api/rpc/json.hpp

```diff
@@ -1044,12 +1044,12 @@ namespace fc::api {
 
     ENCODE(Deadline) {
       Value j{rapidjson::kObjectType};
-      Set(j, "PostSubmissions", v.post_submissions);
+      Set(j, "PartitionsPoSted", v.partitions_posted);
```

Comment:
```suggestion
      Set(j, "PostSubmissions", v.partitions_posted);
```

Reply:
PostSubmissions is an old name for v0 & v2, the new name is PartitionPoSted. Why do you want to keep old name?

Reply:
Api name didn't change